experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.types.ts

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextProps } from '../Text';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
+import { ITextTokens } from '../../tokens/TextTokens';
+import { IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from 'src/tokens';
+import { IPressableState, IPressableProps } from '../Pressable';
+
+export const radioButtonName = 'RadioButton';
+
+// The state of the Radio Button. It extends IPressableState to keep track of press, focus, and hover
+export interface IRadioButtonState extends IPressableState {
+  /*
+   ** Whether or not the RadioButton is currently checked
+   */
+  selected: boolean;
+
+  /*
+   ** Whether or not the option is disabled
+   */
+  disabled: boolean;
```

Comment:
make this property optional?

Reply:
This is state, not a prop. It's always going to be either true or false